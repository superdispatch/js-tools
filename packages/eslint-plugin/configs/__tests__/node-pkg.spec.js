'use strict';

const { testInheritance } = require('../__testutils__/test-eslint-config');

testInheritance('node-pkg', 'node');
