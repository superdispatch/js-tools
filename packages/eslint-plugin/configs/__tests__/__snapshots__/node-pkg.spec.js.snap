// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inheritance config 1`] = `
Snapshot Diff:
Compared values have no visual difference.
`;

exports[`Inheritance dev rules 1`] = `
Snapshot Diff:
Compared values have no visual difference.
`;

exports[`Inheritance rules 1`] = `
Snapshot Diff:
- First value
+ Second value

@@ --- --- @@
    "node/no-unpublished-require": Array [
-     "off",
+     "error",
    ],
`;
